src/inspect_ai/model/_cache.py

```diff
@@ -148,13 +148,16 @@ def __init__(
 
 
 def _cache_key(entry: CacheEntry) -> str:
+    config_dump = entry.config.model_dump(
+        # Exclude fields that don't affect the model output
+        exclude=set(["max_retries", "timeout", "max_connections"])
+    )
+    for k in [*config_dump]:
+        if k.startswith("batch_"):
+            config_dump.pop(k)
```

Comment:
Since we're already explicitly enumerating keys that don't impact the model output, I think it would be way cleaner to just add the additional new batch related params to the list. Plus, as coded, the simple case of `batch=True` and no other config would allow `batch` to the bin the cache key - which I don't think is your goal.

Reply:
> I think it would be way cleaner to just add the additional new batch related params to the list.

We'll forget to update the list if another one gets added -> cache is busted for everyone

> Plus, as coded, the simple case of `batch=True` and no other config would allow `batch` to the bin the cache key

I don't understand what this means.

Reply:
Good point